vendors/twtg/drivers/neon-ds-ts-xx-xx/driver.yaml

```diff
@@ -0,0 +1,16 @@
+# Formal driver name
+name: TWTG Neon Temperature Sensor Driver
+# Simple description of the driver functionalities
+description: A driver that decodes and encodes TWTG Neon Temperature Sensor.
+# Mandatory - id of the company that developed the driver
+# This id will be used to construct the id of the driver (Should be in lowercase, no special characters, maximum 8 characters)
+producerId: twtg
+# Email of the developer
+developerEmail: marten.lootsma@twtg.io
+# In case of a driver per model: `<vendorId>:<modelId>:<modelVersion>`
+# In case of a driver for multiples model: `<vendorId>:generic:<genericVersion>`
+# Needed for linking the model with a specific driver -> must be the same one used in model.yaml in the corresponding model
+protocolId: twtg:ts:1
```

Comment:
To where this 1 corresponds to?

In our binary message we have a protocol version field. Typically our drivers are compatible with all versions in the field. The highest protocol version for the TS is 2. Shouldn't this number be 2 as well? Or is this decoupled? If that is the case, what determines this number?

Reply:
Yes, for all drivers you add in this PR (TS, VS-MT, VS-QT) it is protocol version 2.

For VS-CS (which is not part of this PR) it is protocol version 1

Reply:
Corrected on Temperature Sensor, and MT and QT Valve Sensors